### PR TITLE
fix kubectl get cluster display issue

### LINKED ISSUE #31073: Kubectl get clusters output is missaligned
```
./cluster/kubectl.sh --context=federation-cluster get clusters
NAME       STATUS    VERSION   AGE
cluster1   Unknown   1m
```
cc : @kubernetes/sig-cluster-federation

### PR DESCRIPTION
**Pull request by adohe-zz**
**What this PR does / why we need it**:

**Which issue this PR fixes** _(optional, in `fixes #<issue number>(, #<issue_number>, ...)` format, will close that issue when PR gets merged)_: fixes #31073

**Release note**:
`NONE`

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/31109)

### CODE REVIEW

**Comment by AdoHe** [comment]
@mwielgus add a quick fix to #31073, ptal.

**Comment by dims** [comment]
@AdoHe LGTM 👍

File: pkg/kubectl/resource_printer.go [hunk 1]
```diff
@@ -940,9 +940,10 @@ func printCluster(c *federation.Cluster, w io.Writer, options PrintOptions) erro
 		statuses = append(statuses, "Unknown")
 	}
 
-	if _, err := fmt.Fprintf(w, "%s\t%s\t%s\n",
+	if _, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\n",
 		name,
 		strings.Join(statuses, ","),
+		"",
```
Comment (mwielgus) [inline pkg/kubectl/resource_printer.go]: Something should be printed here.

Comment (mwielgus) [inline pkg/kubectl/resource_printer.go]: Actually we should do the other way round - please remove VERSION column. It should have been removed in https://github.com/kubernetes/kubernetes/pull/28186.

**Comment by dims** [comment]
+1 to fix the underlying issue rather than just the output :)

**Comment by nikhiljindal** [comment]
Yes we should remove the column as @mwielgus suggested.
Thanks @AdoHe for picking this issue.

**Comment by AdoHe** [comment]
Ok, I will do it.

**Comment by AdoHe** [comment]
@nikhiljindal @mwielgus update, ptal.

**Comment by nikhiljindal** [comment]
LGTM, thanks!

**Pull request merged**